dist/rpm/input-leap.spec.in

```diff
@@ -14,46 +14,44 @@ URL: https://github.com/input-leap/input-leap
 Source0: %{tarname}.tar.xz
 
 %if 0%{?fedora}
-BuildRequires: avahi-compat-libdns_sd-devel
-BuildRequires: cmake3
+BuildRequires: pkgconfig(avahi-compat-libdns_sd)
+BuildRequires: cmake3 >= 3
 BuildRequires: desktop-file-utils
 BuildRequires: gcc-c++
-BuildRequires: gmock-devel
-BuildRequires: gtest-devel
-BuildRequires: gulrak-filesystem-devel
-BuildRequires: libX11-devel
-BuildRequires: libXtst-devel libXinerama-devel libXrandr-devel
-BuildRequires: libICE-devel libSM-devel
-BuildRequires: libcurl-devel
-BuildRequires: openssl-devel
-BuildRequires: qt6-qtbase-devel
-BuildRequires: qt6-qttools-devel
-BuildRequires: qt6-qt5compat-devel
+BuildRequires: pkgconfig(gmock)
+BuildRequires: pkgconfig(gtest)
+BuildRequires: cmake(ghc_filesystem)
+BuildRequires: pkgconfig(x11)
+BuildRequires: pkgconfig(xtst) pkgconfig(xinerama) pkgconfig(xrandr)
+BuildRequires: pkgconfig(ice) pkgconfig(sm)
+BuildRequires: pkgconfig(libcurl)
+BuildRequires: pkgconfig(openssl)
+BuildRequires: pkgconfig(Qt6)
+BuildRequires: pkgconfig(Qt6Designer)
+BuildRequires: pkgconfig(Qt6Core5Compat)
 BuildRequires: make
+BuildRequires: pkgconfig(libei-1.0)
+BuildRequires: pkgconfig(libportal) >= 0.8
 %endif
 
 %if 0%{?suse_version}
-BuildRequires: avahi-compat-mDNSResponder-devel
+BuildRequires: pkgconfig(avahi-compat-libdns_sd)
 BuildRequires: cmake >= 3
```

Comment:
So afaict with the switch to `pkgconfig()` the only difference between fedora and suse is now how cmake is installed. And I noticed the [Fedora spec file](https://src.fedoraproject.org/rpms/input-leap/blob/rawhide/f/input-leap.spec#_19) uses 
```
BuildRequires:	cmake >= 3.12
```

So we might be able to get rid of the distinction between fedora and suse completely.

Reply:
With the newest commit, one could also change this to just cmake >= 3 for fedora.
Fedora requires gulrak-filesystem-devel and some other qt libraries, I don't now if this can be dropped

Reply:
iirc input-leap has gulrak-fs as a submodule so what are the chances the suse build uses that because it's missing from the build reqs?

Reply:
I removed it in Fedora, it still works